### PR TITLE
Add configurable stars in background

### PR DESCRIPTION
**Pull request by tobiasbp**

### CODE REVIEW

**Review by ivan-BRHM — COMMENTED** [review]
I personally think we should have the stars fade in and out rather than blinking. And then after this merge we need extensive reformating of the project.

**Comment by tobiasbp** [comment]
> I personally think we should have the stars fade in and out rather than blinking. And then after this merge we need extensive reformating of the project.

I have pushed an update with fading stars. Let me know what you think.

**Review by ivan-BRHM — CHANGES_REQUESTED** [review]
one little thing

File: game_sprites.py [hunk 1]
```diff
@@ -20,21 +21,28 @@ def __init__(self, position: Tuple[int, int], size:int = 10, scale:float=0.5, al
         self.alpha_max = alpha_max
         self.angle = arcade.rand_angle_360_deg()
 
-        self.blink()
+        # Start fade pos between 0 and 2 * pi
+        self.fade_pos = 2 * random() * pi
+        self.fade_speed = random() / 30
```
[the remark this answers is not in the document]
Comment (tobiasbp) [reply game_sprites.py:26]: Yes. I'll polish and put values in the config.

**Pull request merged**